WordPressKit/WordPressKit/RemoteBlogOptionsHelper.m

```diff
@@ -27,6 +27,7 @@ + (NSDictionary *)mapOptionsFromResponse:(NSDictionary *)response
                                           @"gmt_offset",
                                           @"allowed_file_types",
                                           @"frame_nonce",
+                                          @"jetpack_version",
```

Comment:
ha so we had the rest of the code to handle the Jetpack version but we were missing this mapping so effectively it wasn't being pulled in?

Reply:
It was working only for XML-RPC sites, and until now only used on those to check if the version was enough before connecting to WordPress.com